flink-streaming-java/src/test/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironmentTest.java

```diff
@@ -520,9 +518,7 @@ void testConfigureCheckpointStorage() {
                 StreamExecutionEnvironment.getExecutionEnvironment(configuration);
 
         CheckpointStorage storage = env.getCheckpointConfig().getCheckpointStorage();
-        assertThat(storage).isInstanceOf(JobManagerCheckpointStorage.class);
-        assertThat(((JobManagerCheckpointStorage) storage).getCheckpointPath())
-                .isEqualTo(new Path(path));
+        assertThat(storage).isNull();
```

Comment:
This test becomes nonsense. I would remove it entirely as long as there is a test to cover the case that the checkpoint storage set in StreamExecutionEnvironment can be properly created at runtime.

Reply:
Since there are already corresponding unit tests covering the creation of StateBackend, CheckpointStorage, and Restart Strategy from the job configuration of the execution graph, I have added a case, testJobConfigFromEnvToExecutionGraph, to cover the transfer of the configuration from the environment to the execution graph as the job configuration.